stan/math/prim/meta/as_scalar.hpp

```diff
@@ -0,0 +1,59 @@
+#ifndef STAN_MATH_PRIM_SCAL_FUN_AS_SCALAR_HPP
+#define STAN_MATH_PRIM_SCAL_FUN_AS_SCALAR_HPP
```

Comment:
This file was previously prim/mat/meta/as_scalar.hpp. Not sure why its showing at a s new file.

Reply:
Oh and this should be STAN_MATH_PRIM_META_AS_SCALAR_HPP. Not sure why this has FUN in the guard.

Reply:
Don't bother too much with this. It is deleted in GLM broadcasting PR.